flake8_async/visitors/visitors.py

```diff
@@ -93,7 +93,9 @@ def visit_With(self, node: ast.With | ast.AsyncWith):
             var_name = item.optional_vars.id
 
             # check for trio.open_nursery
-            nursery = get_matching_call(item.context_expr, "open_nursery")
+            nursery = get_matching_call(
+                item.context_expr, "open_nursery", base=("trio",)
+            )
```

Comment:
nurseries only exist in trio. Could expand the check to work with anyio taskgroups though